tmt/schemas/plan.yaml

```diff
@@ -172,6 +186,30 @@ properties:
           inherit-environment:
             type: boolean
 
+          adjust-discover:
+            type: string
+            enum: ["+", "append", "=", "replace"]
```

Comment:
Right now we use
https://github.com/teemtee/tmt/blob/e35ba6dd9679ee8661b32424b1f20490a367ce91/tmt/schemas/discover/fmf.yaml#L116-L117
I don't think we have a good way to do the schema validation for fmf patterns, so I think it would be better to skip it like in the part above. The explicit `Step inheritance operators` should also not be needed.